pages/docs/couple-your-code/couple-your-code-defining-mesh-connectivity.md

```diff
@@ -48,52 +54,45 @@ Quads are only supported since v2.1. For older version, the methods only exist a
 The following code shows how mesh connectivity can be defined in our example. For sake of simplification, let's only define one triangle and let's assume that it consists of the first three vertices.
 
 ```cpp
-
-[...]
-
-std::vector<int> vertexIDs(vertexSize);
+/* ... */
+
+// We define the unit square in 2D
+std::vector<double> coords{
+    0, 0, // A
+    0, 1, // B
+    1, 0, // C
+    1, 1  // D
+};
+std::vector<VertexID> vertexIDs(4);
```

Comment:
Just curious: how does the user get access to the `VertexID` data type?

Reply:
It lives in `precice/Types.hpp`, which is included by the `Participant.hpp`.
Does that need an explanation in the example?

Reply:
> Does that need an explanation in the example?

No, I think its fine if it is included by the `Participant.hpp`. Users will probably figure it out.